src/views/domain-page/domain-page-tabs/__tests__/mock-console-error.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Mocks console.error and silences known errors.
+ *
+ * Apply this utility to mute known errors and avoid polluting the test output.
+ */
+export const mockConsoleError = ({
```

Comment:
This can be a shared util in src/test-utils; it can definitely be reused in more tests